data/hai/hai missions.txt

```diff
@@ -1440,11 +1440,10 @@ mission "Strider: Coalition 2"
 
 
 event "pond strider mass production"
-	shipyard "Hai Basics"
-		"Flea"
-	shipyard "Hai Advanced"
-		"Pond Strider"
-		"Flea"
+	shipyard "Mon Ki i'Hiya Export"
+		add "Flea"
+	shipyard "Mon Ki i'Hiya Showroom"
+		add "Pond Strider"
```

Comment:
Since the old outfitters and shipyards will no longer be present at planets, I think doing just means people who have already done this will no longer have access to these items?
You'd need a patch mission that offers if the old version of this event was applied but this one has not been that applies the new version of the event.
I would suggest renaming this event (perhaps just add `updated` or something to the end of the name), but then you should also add an isolated line that is just `event "pond strider mass production"` for compatibility in case a person has a mission calling this event in their savefile.
See the Remnant Gascraft Puffin patch mission and #7633.